npm/src/upload/index.ts

```diff
@@ -74,6 +83,10 @@ export class FileUploader {
   }
 
   initialiseFormListeners(): void {
+    const checkIfJudgmentUserExists: boolean | undefined =
```

Comment:
I'd have it as the ternary. It could return `judgment_user = false`. Its is an implementation detail in Keycloak that we don't set it to `false`, which may change in the future.

Reply:
Cool. thanks.